source/extensions/filters/common/lua/lua.h

```diff
@@ -11,6 +11,8 @@
 #include "common/common/c_smart_ptr.h"
 #include "common/common/logger.h"
 
+#define LUAJIT_ENABLE_GC64
```

Comment:
Is this good? I think you need define them when building LuaJIT, no?

Reply:
That was leftover from experiments. Turns out GC64 doesn't solve the alignment issues, so I'm just going to blacklist ASAN over the entire LuaJIT code base. I'm looking at the zlib issues that remain now.

Reply:
https://github.com/madler/zlib/issues/419